mastering-plone/behaviors_1.md

```diff
@@ -15,13 +15,13 @@ This chapter is about Plone Classic.
 
 Code for the beginning of this chapter:
 
-```
+```shell
 git checkout talks
 ```
 
 Code for the end of this chapter:
 
-```
+```shell
 git checkout behaviors_1
 ```
```

Comment:
```suggestion

{doc}`code`

```

Reply:
Meh. The suggested change does not handle literal backticks well, but you get the idea.

Reply:
I did move the link to code.html to the end only for all chapters that have "get the code" as sidebar title. for the rest of chapters I would prefer to leave them as they are, as the line 
Get the code! ([More info](http://localhost:3002/mastering-plone/code.html))
seems to be fine for me.